pkgs/servers/misc/navidrome/default.nix

```diff
@@ -1,50 +1,61 @@
-{ lib, stdenv, fetchurl, ffmpeg, ffmpegSupport ? true, makeWrapper, nixosTests }:
+{ callPackage, buildGoModule, fetchFromGitHub, lib, pkg-config, stdenv, ffmpeg, taglib, zlib, ffmpegSupport ? true, makeWrapper, nixosTests }:
```

Comment:
And maybe expand this to one per line for readability, improved diffs and merges.

Reply:
Changed to one per line